go/docker/build.sh

```diff
@@ -0,0 +1,17 @@
+cat > ./Dockerfile << EOF
```

Comment:
I think we can use `golang:1.8` as the base Docker image so that it will reduce the Image size.